libbeat/docs/command-reference.asciidoc

```diff
@@ -113,6 +113,30 @@ template.
 Exports the current configuration to stdout. If you use the `-c` flag, this
 command exports the configuration that's defined in the specified file.
 
+
+
+*`dashboard`*::
+Exporting a dashboard allows to store a dashboard on disk in a
+module and load it automatically. The following command can be used:
++
+["source","shell",subs="attributes"]
+----
+$ {beatname_lc} export dashboard --id="dashboard-id" > dashboard.json
+----
++
+The `dashboard-id` can be found in the Kibana URL. By default `export dashboard`
+will write the dashboard to stdout. Above it's written into `dashboard.json` so
+it can later imported again. The file contains the dashboard with all
+visualizations and searches. The index pattern is removed as it is
+expected to be loaded separately for {beatname_uc}.
++
+The generated `dashboard.json` file can be copied into the `kibana/6/dashboard`
+directory of {beatname_lc} and next time `{beatname_lc} setup dashboards` is
```

Comment:
@ruflin  in order for the attribute to resolve
```
`{beatname_lc} setup dashboards`
```
 needs to be
````
+{beatname_lc} setup dashboards+
```
Sorry for missing *another* problem. Sigh.

Reply:
Changed. 

Why does the tick not work here and have to use `+` instead?

Reply:
asciidoc treats everything within back ticks as text. Using plus signs forces asciidoc to evaluate the asciidoc references first.